hardware/MQTT.cpp

```diff
@@ -3098,12 +3148,12 @@ void MQTT::InsertUpdateSwitch(_tMQTTASensor* pSensor)
 		{
 			if (root["brightness"].empty() && root["position"].empty())
 			{
-#ifdef _DEBUG
-				_log.Debug(DEBUG_NORM, "ERROR: Last Payload is missing state field (%s/%s)", pSensor->unique_id.c_str(), szDeviceName.c_str());
```

Comment:
I remember that we did this on purpose as else some users would have continues error lines because some hardware sends wrong MQTT messages
You will see them when you compile/test in debug anyway
So maybe best to revert this change ?

Reply:
Same here, I definitly did not touch this line. Is it possible that it was merged somehow when I checked it in? I am never really sure of what I do when using Github... Anyway, I will revert this.